pdks/FinFET14nm_Mock_PDK/Align_primitives.py

```diff
@@ -0,0 +1,42 @@
+import argparse
+import pathlib
+import logging
+import json
+
+from align.cell_fabric import generate_primitive
+
+def main( args):
+    logging.basicConfig(level=logging.getLevelName(args.logLevel))
+    return generate_primitive(
```

Comment:
Same comment here. Use keywords or pass args into "generate_primitive".

Reply:
Align_primitives.py is no longer used in the make flow and should be eventually removed. Leaving this as is for now since it still gets used in PDK tests.

Reply:
Created issue #278 to track this.